.github/tailor.yaml

```diff
@@ -0,0 +1,10 @@
+rules:
+  - name: commit title
+    description: all commit titles must have a scope, be lowercase, and be no more than 50 characters
+    expression:  |-
+      .commits all((.message test "^([a-z/]+)|(\\*): [a-z \\-'.]+") and (.message test "^.{0,50}"))
+
+  - name: commit body
+    description: all commits must have a message body with each line having no more than 70 characters
+    expression: |-
+      .commits all(.message test "^.*\\n(\\n.{0,70})+$")
```

Comment:
Won't this regex allow a commit message like so (avoiding the required body):

```
example: commit message


```

Should probably be `{1,70}`

Reply:
It will. I left it at {0,70} to allow for empty lines. Let me try one more expression.